pkg/registry/core/pod/storage/eviction.go

```diff
@@ -194,10 +195,8 @@ func (r *EvictionREST) Create(ctx context.Context, name string, obj runtime.Obje
 		// we cannot ignored the PDB for this pod, so this is the fall through case.
 	}
 
```

Comment:
if it is legitimate to ignore the PDB for not-ready pods, would it be simpler to update `canIgnorePDB` and `shouldEnforceResourceVersion` to return true for not-ready pods and let the above loop handle this?

Reply:
yeah, that was the update from two minutes ago